src/api/payment-requests/payment-requests.md

```diff
@@ -796,6 +796,72 @@ Alternatively you can provide an external transaction Id and the Centrapay [Asse
 | 403    | {% break _ REQUEST_PAID %}    | The cancellation cannot be completed because the request `status` is `paid`.            |
 
+<a name="void">
+### Void a Payment Request **EXPERIMENTAL**
+
+Voiding a payment request will cancel the request and trigger any refunds if necessary.
+
+{% reqspec %}
+  POST '/api/payment-requests/{paymentRequestId}/void'
+  auth 'jwt'
+  path_param 'paymentRequestId', 'MhocUmpxxmgdHjr7DgKoKw'
+{% endreqspec %}
+
+{% h4 Example response payload when payment request is unpaid%}
+{% json %}
+{
+  "type": "cancel",
+	"cancellationReason": "CANCELLED_BY_MERCHANT",
+  "value": {
+    "currency": "NZD",
+    "amount": "1000",
+  },
+  "assetType": "centrapay.nzd.main",
+  "paymentRequestId": "MhocUmpxxmgdHjr7DgKoKw",
+  "shortCode": "CP-C7F-ZS5-032",
+  "merchantName": "Centrapay Café",
+  "merchantId": "26d3Cp3rJmbMHnuNJmks2N",
+  "merchantAccountId": "C4QnjXvj8At6SMsEN4LRi9",
+  "merchantConfigId": "5efbe2fb96c08357bb2b9242",
+  "createdAt": "2021-06-08T04:04:27.426Z",
+  "createdBy": "crn::user:0af834c8-1110-11ec-9072-3e22fb52e878",
+  "paymentRequestCreatedBy": "crn::user:0af834c8-1110-11ec-9072-3e22fb52e878",
+  "activityNumber": "2",
+}
+{% endjson %}
+
+{% h4 Example response payload when payment request is paid%}
+{% json %}
+{
+  "type": "refund",
+  "value": {
+    "currency": "NZD",
+    "amount": "1000",
+  },
+  "assetType": "centrapay.nzd.main",
+  "paymentRequestId": "MhocUmpxxmgdHjr7DgKoKw",
+  "shortCode": "CP-C7F-ZS5-032",
+  "merchantName": "Centrapay Café",
+  "merchantId": "26d3Cp3rJmbMHnuNJmks2N",
+  "merchantAccountId": "C4QnjXvj8At6SMsEN4LRi9",
+  "merchantConfigId": "5efbe2fb96c08357bb2b9242",
+  "createdAt": "2021-06-08T04:04:27.426Z",
+  "createdBy": "crn::user:0af834c8-1110-11ec-9072-3e22fb52e878",
+  "paymentRequestCreatedBy": "crn::user:0af834c8-1110-11ec-9072-3e22fb52e878",
+  "activityNumber": "3",
+},
+{% endjson %}
+
+{% h4 Error Responses %}
+
+| Status |                Code                |                                                                                                                                 Description                                                                                                                                 |
+| :----- | :--------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 403    | {% break _ VOID_WINDOW_EXCEEDED %} | The void window is closed 24 hours after the request's `createdAt`. If the request is paid, the [Refund][] endpoint can be used to reverse the payment after the void window has closed.                                                                                    |
```

Comment:
This sentence could be tidied up a bit something like
``The void window lasts for 24 hours after the payment request has been created. If the request has been paid, the [Refund][] endpoint can be used to reverse the payment.``